src/app/search-employee/search-employee.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="container">
+    <mat-card class="search-form-container">
+        <mat-card-title>Search for an Employee</mat-card-title>
+        <form>
+            <mat-form-field class="user-id">
+                <mat-label>Employee ID</mat-label>
+                <input 
+                matInput 
+                placeholder="0123456" 
+                name="employeeid" 
+                [(ngModel)]="employeeQuery.employeeid" 
+                [formControl]="employeeid" 
+                [errorStateMatcher]="matcher"
+                (change)="isUserIdValid()">
```

Comment:
A employeeid is a uuidv4 string. If a user enters this, they are looking for an exact match. You dont need to validate it. I made it so that if an invalid id is sent to the backend, the id is ignored

Reply:
It does not hurt anything to validate it does it? And it will inform the user of their mistake instead of ignoring it.

Reply:
At a minimum I'll update this to specify an employeeid instead of a user ID

Reply:
i guess what i am trying to say that you would have to change the function to validate that what they entered is actually a uuid

Reply:
You're right. I'll update it. If I can find a regex or something for UUIDv4 I'll use that, otherwise I'll skip validation.